Podfile

```diff
@@ -175,7 +175,7 @@ abstract_target 'Apps' do
   ## Gutenberg (React Native)
   ## =====================
   ##
-  gutenberg tag: 'v1.83.0'
+  gutenberg commit: '20d7e4c6eba8d46853379ce1bf4b061c61f22b83'
```

Comment:
Friendly reminder we'll need to update this to the release tag before merging.

Reply:
Reference updated to release tag in https://github.com/wordpress-mobile/WordPress-iOS/pull/19462/commits/586e2adf32ae6cdda425728bd5d909e5d5c72cfe.